src/main/java/lotto/domain/WinningLotto.java

```diff
@@ -26,11 +50,15 @@ private static List<Integer> parseWinningNumbers(String winningNumbers) {
                     winningNumbers.replaceAll("\\s", "").split(",")
             ).mapToInt(Integer::parseInt).boxed().collect(Collectors.toList());
         } catch (NumberFormatException e) {
-            throw new IllegalArgumentException(ErrorMessage.LOTTO_NUMBER_NUMERIC_ERROR);
+            throw new IllegalArgumentException(ErrorMessage.LOTTO_NUMBER_NUMERIC_ERROR.getMessage());
         }
     }
 
     public Integer compareLotto(Lotto comparedLotto) {
         return (int) this.winningLotto.stream().filter(comparedLotto.getLotto()::contains).count();
     }
+
+    public Boolean isBonusIncluded(Lotto comparedLotto) {
+        return comparedLotto.getLotto().contains(bonusNumber);
```

Comment:
현재 객체 내부값을 꺼내어 직접 연산을 하고 있습니다. 이렇게되면 private로 변수의 접근제어자를 설정한 이유가 사라지는 것 같아요. 

```suggestion
        return comparedLotto.containNumber(bonusNumber);
```

이런식으로 요청과 응답만 오고가도록 OOP 스럽게 작성할 수 없을까요?